yt_dlp/extractor/facebook.py

```diff
@@ -524,6 +530,10 @@ def extract_metadata(webpage):
                     webpage, 'view count', default=None)),
                 'concurrent_view_count': get_first(post, (
                     ('video', (..., ..., 'attachments', ..., 'media')), 'liveViewerCount', {int_or_none})),
+                'unified_reactors_count': post_stats.get('unified_reactors', {}).get('count'),
+                'total_comment_count': post_stats.get('total_comment_count'),
+                'share_count': parse_count(post_stats.get('share_count_reduced')),
+                'owner': owner,
```

Comment:
Those are not valid info dicts values. Please have a look for what values are allowed to be returned here: https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py#L104

I'm not sure what `unified_reactors_count` means, are those effectively the likes? Then `like_count` could be used.
`total_comment_count` should be `comment_count`. `share_count` could be `repost_count`.
I am not sure what `owner` in this context means. How is it different from `uploader`?
Though currently, the uploader and title value seem to both be extracted incorrectly.

Reply:
Thanks @seproDev for your feedback. I was unaware of the existing fields you mentioned. Let me make the changes now.